types/CustomPrice.md

```diff
@@ -0,0 +1,99 @@
+# Spot Price Feeds
+
+## Query Name
+
+- `CustomPrice`
+
+## Query Description
+
+This query returns the price of a non-crypto asset like a stock, derivatives product, or legacy market commodity.
+
+## Query Parameters
+
+The `CustomPrice` query has three parameters. The identifier parameter provides context for the data. The asset and currencty parameters specify the pricing pair.
+
+1. **identifier**
```

Comment:
Maybe we should include some more example identifiers, such as: ForexPrice, IndexPrice, PreciousMetalPrice